### PR TITLE
React router

### PR DESCRIPTION
**Pull request by jamaljeantobias**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-router": "^4.2.0"
+    "react-router": "^4.2.0",
```
Comment (aconanlai) [inline package.json]: shouldn't need this, react-router-dom is what we use

File: src/App.css [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
-  color: white;
-}
-
-.App-title {
-  font-size: 1.5em;
-}
 
-.App-intro {
-  font-size: large;
+html {
```
Comment (aconanlai) [inline src/App.css]: bad practice to style on the actual <html> tag, usually you want to make like a .wrapper div, etc

File: src/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+.dog {
+  background-color: rgb(197, 121, 0);
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  height: 100px;
+  width: 100%;
+  overflow:hidden;
+  align-content: center;
```
Comment (aconanlai) [inline src/Footer/Footer.css]: have you confirmed that align-content here works? it's supposed to only work on a flex container

File: src/Footer/Footer.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+.dog {
+  background-color: rgb(197, 121, 0);
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  height: 100px;
+  width: 100%;
+  overflow:hidden;
```
Comment (aconanlai) [inline src/Footer/Footer.css]: inconsistent spacing after CSS property [no spacing after overflow:]

File: src/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+const Footer = ({ props }) => {
+  //   <NavLink activeClassName="active" to={'/about'} >
+  //   About us
+  // </NavLink>
+
+  // Test Renders - please disregard these
+
+  return (
+    <div classname="dog">
+
```
Comment (aconanlai) [inline src/Footer/Footer.js]: extra blank lines

File: src/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+const Footer = ({ props }) => {
+  //   <NavLink activeClassName="active" to={'/about'} >
+  //   About us
+  // </NavLink>
+
+  // Test Renders - please disregard these
+
+  return (
+    <div classname="dog">
```
Comment (aconanlai) [inline src/Footer/Footer.js]: classname should be className

File: src/Footer/Footer.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+      <h1> Music Player</h1>
```
Comment (aconanlai) [inline src/Footer/Footer.js]: extra space before Music

File: src/Footer/navbar.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+import { Link, NavLink } from 'react-router-dom';
+
+const Nav = ({ props }) => {
+
+  // Test Renders - please disregard these
+
+
+
+  return (
+    <ul classname='navBar'>
```
Comment (aconanlai) [inline src/Footer/navbar.js]: JSX properties should be surrounded by double quotes i.e `className="navBar"` also missed camel case on className

File: src/App.css (hunk 2 above)
Comment (jamaljeantobias) [reply src/App.css]: Ok.

File: src/Footer/Footer.js (hunk 6 above)
Comment (jamaljeantobias) [reply src/Footer/Footer.js]: Yep

File: src/Info.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+
+//
+const Info = () => {
+  return (
```
Comment (aconanlai) [inline src/Info.js]: you don't need an explicit return in using this syntax [stateless functional components]:
https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc

File: src/index.js [hunk 10]
```diff
@@ -2,7 +2,29 @@ import React from 'react';
...
+
+import notFound from './notFound.js'
+import { ReactRouter, BrowserRouter, Route, Switch, Router } from 'react-router-dom'
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+
+
+const ROOT = () => {
```
Comment (aconanlai) [inline src/index.js]: why is this all caps?

**Comment by aconanlai** [comment]
also inconsistent capitalization on component files/folder names, i.e list, etc not capitalized

File: package.json [hunk 11]
```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "eslint-config-airbnb": "^16.1.0",
```
Comment (aconanlai) [inline package.json]: shouldn't be in dependencies

File: src/components/Podcast/Podcast.js [hunk 12]
```diff
@@ -0,0 +1,11 @@
+import React  from 'react';
+
+const Podcast = ({items}) => {
+  console.log(items)
```
Comment (aconanlai) [inline src/components/Podcast/Podcast.js]: forgotten console.log

File: src/index.js [hunk 13]
```diff
@@ -2,7 +2,27 @@ import React from 'react';
...
+
+import {BrowserRouter, Route, Switch } from 'react-router-dom'
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+const Root = () => {
+
+  const Info = () => (
+    <div>BLAAAAH</div>
```
Comment (aconanlai) [inline src/index.js]: `Info` definition should go outside of `Root`, since right now every  time Root is being rendered, you're redefining Info

**Pull request merged**